compiler/dyno/include/chpl/uast/Function.h

```diff
@@ -372,8 +377,11 @@ class Function final : public NamedDecl {
     assert(b != nullptr);
     return b->stmt(i);
   }
-};
 
+  static const char* returnIntentToString(ReturnIntent intent);
```

Comment:
I personally think it's OK to have these things return `const char*` when each enum converts to a fixed length string. E.g., within dyno I'd expect we could have chpldoc call it, (later, possibly error messages).

I'm actually in the opposite camp in that I'd find a `std::string` to be more reprehensible, because IMO it's more of a heavyweight type than is actually needed.

W.r.t. the production compiler, I wouldn't expect this to be called there because it's a different IR at that point (I know we have plans to be able to bridge that gap and inspect using dyno but I don't think we're at that point yet).

Reply:
I think I'd prefer that this method, and others like it, return a ``std::string``. I wondered at first if the null case was useful, but we can always create an exhaustive switch-statement for an enum like this.

Anyways, I think we could address this in a follow-up PR.

Reply:
Adjusted to return `std::string`.